### DISCUSSION TITLE
How can the ades process name be customized?

### DISCUSSION THREAD

**Discussion by totycro** [EOEPCA/eoepca#84] [ADES (Processing)]
When deploying the snuggs process with version 0.3.0, ades creates this process name: `snuggs-0_3_0`.

In our use case, we would like to use the process name in further systems, which however don't allow an underscore in the name. Is there a way to change the format here, or even turn this behavior off, such that the process name is only `snuggs` in ades?

**Comment by rconway** [comment]
The ADES relies upon this naming convention to ensure uniqueness of process names - i.e. including the version in the name in order to support multiple versions of the same app being deployed.
One possibility would be to make the format of this configurable via a template-able parameter.
@bbrauzzi Any thought?
